meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor

```diff
@@ -84,6 +84,9 @@ else
     error_handler $LINENO "no container engine detected"
 fi
 
+tag=""
```

Comment:
Why do you need these? I think we can drop them as they will get assigned in both if branches.